specification/enrich/stats/types.ts

```diff
@@ -38,6 +38,10 @@ export class CacheStats {
   node_id: Id
   count: integer
   hits: integer
+  /* The amount of time in milliseconds spent fetching data from the cache on successful cache hits only. */
+  hits_time_in_millis: long
```

Comment:
We might want to use `DurationValue<UnitMillis>` here to carry additional semantics.

Reply:
Thanks, done in [`1e187ac` (#2845)](https://github.com/elastic/elasticsearch-specification/pull/2845/commits/1e187ac8cbda1e327d1da97ed067d2ed7e98f415).